src/tests/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests/Features/IlrStartDateChanged.feature

```diff
@@ -5,36 +5,50 @@ Feature: IlrStartDateChanged
 
 Scenario: Learner data contains a new start date within the parameters of the incentive scheme
 	Given an apprenticeship incentive exists
-	When the learner data is refreshed with a new valid start date for the apprenticeship incentive
+	When the learner data is updated with new valid start date for the apprenticeship incentive
+	And the incentive learner data is refreshed
 	Then the actual start date is updated
 	And the pending payments are recalculated for the apprenticeship incentive
 	And the learner data is subsequently refreshed
 
 Scenario: Learner data contains a new start date outside of the parameters of the incentive scheme
 	Given an apprenticeship incentive exists
-	When the learner data is refreshed with a new invalid start date for the apprenticeship incentive
+	When the learner data is updated with new invalid start date for the apprenticeship incentive
+	And the incentive learner data is refreshed
 	Then the actual start date is updated
 	And the existing pending payments are removed
 
 Scenario: Clawbacks - Start Date Change Of Circumstance with eligible start date
 	Given an apprenticeship incentive exists
 	And an earning has been paid for an apprenticeship incentive application
-	When the learner data is refreshed with a new valid start date for the apprenticeship incentive
+	When the learner data is updated with new valid start date for the apprenticeship incentive
+	And the incentive learner data is refreshed
 	Then earnings are recalculated
 	And the paid earning is marked as requiring a clawback
-	And earnings are recalculated
 	And a new pending first payment record is created
 	And a new pending second payment record is created
 
 Scenario: Clawbacks - Start Date Change Of Circumstance with ineligible start date
 	Given an apprenticeship incentive exists
 	And an earning has been paid for an apprenticeship incentive application
-	When the learner data is refreshed with a new invalid start date for the apprenticeship incentive
+	When the learner data is updated with new invalid start date for the apprenticeship incentive
```

Comment:
This should be part of the given since it is a precondition of what is being tested which is the learner data refresh